src/antimeridian/_implementation.py

```diff
@@ -269,11 +324,19 @@ def segment_polygon(polygon: Polygon) -> List[List[Point]]:
 
 
 def fix_polygon_to_list(
-    polygon: Polygon, *, force_north_pole: bool, force_south_pole: bool
+    polygon: Polygon,
+    *,
+    force_north_pole: bool,
+    force_south_pole: bool,
+    fix_winding: bool,
 ) -> List[Polygon]:
     segments = segment(list(polygon.exterior.coords))
     if not segments:
-        return [polygon]
+        if fix_winding and not shapely.is_ccw(polygon.exterior):
```

Comment:
Should we also detect incorrectly wound holes as a trigger for calling `orient`? If there are incorrectly wound (CCW) holes, does that cause problems with the segments?

Reply:
@pjhartzell after our offline conversation, I got it working, I think.